### ISSUE TITLE
Are proposals same for all the video finishing training?

### ISSUE THREAD

**Issue by barry2025** [xiaoneil/LPNet#2]
in source code, I see the proposal is initialized randomly and is independent with video (except video length), does that mean the N proposals are the same for all the videos (except length)?

**Comment by xiaoneil** [comment]
Yes, the N proposals are the same for all videos in the same dataset. The model basically learned the statistical information of the most likely location of segments.
